libs/execution_context/EBPF_NATIVE.H

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "ebpf_core_structs.h"
+#include "ebpf_maps.h"
+#include "bpf2c.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+    typedef struct _ebpf_native ebpf_native_t;
+
+    /**
+     * @brief Initialize the eBPF native code module.
+     *
+     * @retval EBPF_SUCCESS The operation was successful.
+     * @retval EBPF_NO_MEMORY Unable to allocate resources for this
+     *  operation.
+     */
+    ebpf_result_t
+    ebpf_native_initiate();
+
+    /**
+     * @brief Uninitialize the eBPF native code module.
+     *
+     */
+    void
+    ebpf_native_terminate();
+
+    /**
+     * @brief Load native module driver and bind to the native module.
+     *
+     * @param[in] service_name Name of the service for the native module.
```

Comment:
Null terminated or not?

Reply:
It is not null terminated. Updated the SAL annotation.